apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -4819,13 +4813,13 @@ label {
     margin-left: 255px;
 }
 
-.grp-col::-webkit-scrollbar-track {
-    background: $whisper-light;
+.grp_col::-webkit-scrollbar-track {
```

Comment:
This change is reverted while according to the code design, we've fixed most of the camel case classes. @prateek-giddh

Reply:
Done